src/commands/apphosting-builds-create.ts

```diff
@@ -16,12 +16,12 @@ export const command = new Command("apphosting:builds:create <backendId>")
     const buildId =
       (options.buildId as string) ||
       (await apphosting.getNextRolloutId(projectId, location, backendId));
-    const branch = options.branch as string;
+    const branch = (options.branch as string | undefined) ?? "main";
```

Comment:
A really neat improvement to this would be to see if they have a RolloutPolicy and pull the "default" branch from there. But that can come later.

Reply:
File a bug and assign it to me. I agree that would be much better, but IIRC rollouts is still an "internal tool" that hasn't gone through API review.